CMakeLists.txt

```diff
@@ -79,6 +79,11 @@ file(RELATIVE_PATH RELATIVE_LIBDIR
 
 message(STATUS "CMAKE_INSTALL_LIBDIR ${CMAKE_INSTALL_LIBDIR}")
 
+set(SHAPELIB_PC ${CMAKE_CURRENT_BINARY_DIR}/shapelib.pc)
```

Comment:
CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_INCLUDEDIR may be sometimes absolute paths. You may take inspiration from https://github.com/OSGeo/libgeotiff/blob/master/libgeotiff/CMakeLists.txt#L365 to https://github.com/OSGeo/libgeotiff/blob/master/libgeotiff/CMakeLists.txt#L380 , and https://github.com/OSGeo/libgeotiff/blob/master/libgeotiff/libgeotiff.pc.in to have support for that use case

Reply:
Addressed by 86a385d5fd28387539cf9cf8a328fed6368236c3.